pt/controllers/StatusControllerTest.java

```diff
@@ -50,22 +50,22 @@ public void showsVersion() throws Exception {
         run(app(), StatusController.class, controller -> {
             final Result result = controller.version();
             assertThat(result.status()).isEqualTo(Http.Status.OK);
-            assertThat(result.contentType()).isEqualTo(Http.MimeTypes.JSON);
+            assertThat(result.contentType()).contains(Http.MimeTypes.JSON);
             assertThat(contentAsString(result)).contains("version").contains("build");
         });
     }
 
-    private void rendersHealthyCtp(final StatusController controller) throws IOException {
-        final Result result = controller.health().get(ALLOWED_TIMEOUT);
+    private void rendersHealthyCtp(final StatusController controller) throws Exception {
+        final Result result = controller.health().toCompletableFuture().get(1, TimeUnit.SECONDS);
         assertThat(result.status()).isEqualTo(Http.Status.OK);
-        assertThat(result.contentType()).isEqualTo(Http.MimeTypes.JSON);
+        assertThat(result.contentType()).contains(Http.MimeTypes.JSON);
         assertThat(contentAsString(result)).contains("\"healthy\" : true");
     }
 
-    private void rendersUnhealthyCtp(final StatusController controller) throws IOException {
-        final Result result = controller.health().get(ALLOWED_TIMEOUT);
+    private void rendersUnhealthyCtp(final StatusController controller) throws Exception {
+        final Result result = controller.health().toCompletableFuture().get(1, TimeUnit.SECONDS);
```

Comment:
join, has no checked exceptions, but here with timeout makes sure the app does not hang